Source/WebCore/platform/graphics/Image.h

```diff
@@ -135,7 +134,7 @@ class Image : public RefCounted<Image> {
     virtual String filenameExtension() const { return String(); } // null string if unknown
     virtual String accessibilityDescription() const { return String(); } // null string if unknown
 
-    virtual void destroyDecodedData(bool destroyAll = true);
+    virtual void destroyDecodedData(bool = true) { }
```

Comment:
I prefer to keep the parameter name, commented out:
`virtual void destroyDecodedData(bool /* destroyAll  */ = true) { }`